proto/path/qos/jsonrpc_validation_error.proto

```diff
@@ -0,0 +1,30 @@
+// TODO_TECHDEBT(@adshmh): Use the JsonRpcResponseValidationError in all JSONRPC QoS.
+// Include this refactor in PR #253 (JUDGE framework)
+//
+// TODO_TECHDEBT(@adshmh): The JsonRpcResponseValidationError should only be constructed by the qos/jsonrpc package.
+// See the comment in qos/jsonrpc/observation.go
```

Comment:
This sort of thing is very helpful.

I know YOU know, but I (and everyone else) doesn't.

Ty.